dropwizard-core/src/main/java/io/dropwizard/setup/Bootstrap.java

```diff
@@ -128,16 +126,6 @@ public void setClassLoader(ClassLoader classLoader) {
         this.classLoader = classLoader;
     }
 
-    /**
-     * Adds the given bundle to the bootstrap.
-     *
-     * @param bundle a {@link Bundle}
-     */
-    public void addBundle(Bundle bundle) {
```

Comment:
I think removing the `Bundle` hooks in `Bootstrap` is apt to surprise people who expect their existing bundles to work after a minor version bump. Would you be opposed to holding off on removing this until 2.0.0?

Reply:
They will still work because `Bundle` implements `ConfiguredBundle<Configuration>`.

The change should be completely backwards-compatible (also see `io.dropwizard.BundleTest`).